sdk/template/Azure.Template/.content/samples/README.md

```diff
@@ -0,0 +1,14 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+# Including relevant stubs from https://review.docs.microsoft.com/help/contribute/metadata-taxonomies#product
+- azure
+name: Azure.Template samples for .NET
+description: Samples for the Azure.Template client library.
+---
+
+# Azure.ServiceTemplate.Template Samples
+
+<!-- please refer to <https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/samples/README.md> to write sample readme. -->
```

Comment:
Isn't that this file?

Reply:
No. All files under .content are skeleton files to replace the files in `sdk\template\Azure.Template` when generate sdk package via `dotnet new`.  They will show in the generated packages for the service except Azure.Template. I updated to use  placeholder for this.